hack/generate-readme.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+TASK=$1
+if [ ! -f "$TASK" ]; then
+  echo "Usage: $0 \$PATH_TO_TASK"
+  exit 1
+fi
+echo "# $(yq '.metadata.name' $TASK) task"
+echo
+yq '.spec.description' $TASK
+echo
+PARAMS=$(yq '.spec.params.[] | ("|" + .name + "|" + (.description // "" | sub("\n", " ")) + "|" + (.default // "") + "|" + (.default != "*") + "|")' $TASK)
```

Comment:
when required == true and default value is an empty string vs when required == false, in both the cases the default value field in the table is blank. Will it be good to add `""` in case it is empty string?

Reply:
hard task, tried to do with yq but not successfully, so added 'sed' which does this magic.